devtools/packaging/scripts/osx/build.sh

```diff
@@ -37,7 +38,7 @@ fi
 mkdir build
 cd build
 cmake ../openmm $CMAKE_FLAGS
-make -j4 all DoxygenApiDocs sphinxpdf
+make -j4 all PythonInstall C++ApiDocs PythonApiDocs sphinxpdf
```

Comment:
If you just try running these scripts interactively, it's very obvious what's going on. `make PythonInstall` requires `make install`, because distutils picks links `libOpenMM.{so,dylib}` from `CMAKE_INSTALL_PREFIX` (I changed this requirement #1320, but that's not merged). And `make PythonApiDocs` requires `make PythonInstall`, because it imports OpenMM to read the docsstrings to build the documentation.